weblate/formats/ttkit.py

```diff
@@ -438,7 +438,7 @@ def is_valid_base_for_new(
         except Exception as exception:
             if errors is not None:
                 errors.append(exception)
-            report_error(cause="File parse error")
+            report_error(cause="File-parsing error")
```

Comment:
@nijel is it possible to change this? Isn’t it gonna break something?

Reply:
Yes, there is no problem with this.